src/components/Clickable/Clickable.tsx

```diff
@@ -1,33 +1,25 @@
 /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
-import React, { AriaRole, forwardRef, useRef } from "react";
+import React, { AriaRole, forwardRef } from "react";
 import NOOP from "lodash/noop";
-import cx from "classnames";
-import useMergeRefs from "../../hooks/useMergeRefs";
-import { BEMClass } from "../../helpers/bem-helper";
-import { useKeyboardButtonPressedFunc } from "../../hooks/useKeyboardButtonPressedFunc";
-import { getTestId } from "../../tests/test-ids-utils";
 import { ComponentDefaultTestId } from "../../tests/constants";
 import VibeComponentProps from "../../types/VibeComponentProps";
 import VibeComponent from "../../types/VibeComponent";
-import "./Clickable.scss";
-
-const CSS_BASE_CLASS = "monday-style-clickable";
-const bemHelper = BEMClass(CSS_BASE_CLASS);
+import useClickable from "../../hooks/useClickable";
 
 export interface ClickableProps extends VibeComponentProps {
   elementType?: keyof JSX.IntrinsicElements | string;
   className?: string;
   id?: string;
   children?: React.ReactNode;
   role?: AriaRole;
-  onClick?: (event: React.MouseEvent) => void;
+  onClick?: (event: React.MouseEvent | React.KeyboardEvent) => void;
   enableTextSelection?: boolean;
   onMouseDown?: (event: React.MouseEvent) => void;
   ariaLabel?: string;
   ariaHidden?: boolean;
-  ariaHasPopup?: boolean | string;
+  ariaHasPopup?: boolean;
   ariaExpanded?: boolean;
-  tabIndex?: string;
+  tabIndex?: string | number;
```

Comment:
decided not to do breaking change here, thought tabIndex should be a number - converting it to number afterwards

Reply:
let's add a note for remove string on vibe 2.0 ?

Reply:
note added